recipes/zimg/all/conanfile.py

```diff
@@ -0,0 +1,136 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, MSBuild, tools
+from conans.errors import ConanInvalidConfiguration
+import glob
+import os
+
+
+class ZimgConan(ConanFile):
+    name = "zimg"
+    description = "Scaling, colorspace conversion, and dithering library"
+    topics = ("conan", "zimg", "image", "manipulation")
+    homepage = "https://github.com/sekrit-twc/zimg"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "WTFPL"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    exports_sources = "patches/**"
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if not self.settings.build_type:
+            self.settings.build_type = "Release"
+        if self.settings.build_type not in ("Release", "Debug"):
```

Comment:
Now I see, you have to validate a string pattern ...

Reply:
When I look at it now, is the explicit setting really required?

Reply:
You could use `if str(self.settings.build_type) not in ("Release", "Debug"):` instead

Reply:
> When I look at it now, is the explicit setting really required?

The explicit setting is required, we haven't had time to implement the profile iteration in order yet. If the setting is not given it would be built in `Debug` 😞

Reply:
By explicit settings, I meant doing `settings.build_type = "Release"` above.
I'll remove these lines, as @jgsogo suggested.